pkg/kubelet/dockertools/docker.go

```diff
@@ -454,7 +454,7 @@ func GetDockerPodInfo(client DockerInterface, manifest api.PodSpec, podFullName
 	for _, container := range manifest.Containers {
 		expectedContainers[container.Name] = container
 	}
-	expectedContainers["net"] = api.Container{}
+	expectedContainers["POD"] = api.Container{}
```

Comment:
Introduced a constant with a comment saying it should match kubelet constant. Let me know if that works otherwise build is failing with import cycle.